Samples/excel-data-types-explorer/src/taskpane/taskpane.html

```diff
@@ -0,0 +1,56 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. -->
+<!-- This file shows how to design a first-run page that provides a welcome screen to the user about the features of the add-in. -->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Office Add-ins Excel Data Types Explorer</title>
+
+    <!-- Office Preview JavaScript API -->
+    <script type="text/javascript" src="https://appsforoffice.microsoft.com/lib/beta/hosted/office.js"></script>
+
+    <!-- For more information on Fluent UI, visit https://developer.microsoft.com/fluentui#/. -->
+    <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css"/>
+
+    <!-- Template styles -->
+    <link href="taskpane.css" rel="stylesheet" type="text/css" />
+    <script src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.3.1.js"></script>
+</head>
+
+<body class="ms-font-m ms-welcome ms-Fabric">
+    <section class="ms-font-m main">
+        <h1>Create and explore data types</h1>
+        <p>Use this tool to create data types or to explore data types that already exist in a workbook.</p>
+        <p>To create a data type, use the <strong>Data type</strong> select list to choose a data type (string, double, boolean, entity, web image, or formatted number), enter data in the displayed fields, and then select the <strong>Set data</strong> button. The data type will be inserted into the currently active cell on the worksheet.</p>
```

Comment:
Are these data types in any particular order? Entity seems like the "cool" one, but it's buried in the middle of the list.